src/WarcraftLegacies.Source/Quests/Fel Horde/QuestHellfireCitadel.cs

```diff
@@ -4,18 +4,25 @@
 using MacroTools.FactionSystem;
 using MacroTools.QuestSystem;
 using MacroTools.QuestSystem.UtilityStructs;
-using MacroTools.Wrappers;
 using WCSharp.Shared.Data;
 using static War3Api.Common;
 
 namespace WarcraftLegacies.Source.Quests.Fel_Horde
 {
+  /// <summary>
+  /// Drive the Draenai out of Outland, take control of various control points in outland and upgrade main to t3 in order to unlock Hellfire Citadel.
+  /// </summary>
   public sealed class QuestHellfireCitadel : QuestData
   {
     private readonly List<unit> _demonGates;
     private readonly List<unit> _rescueUnits = new();
 
-    //Todo: mediocre flavour
```

Comment:
Let's leave these todo comment in. At some point I want to go and fix up these quests with overly generic flavour.